typescript/src/schema/api.ts

```diff
@@ -97,6 +99,7 @@ export class API {
     // users specify the actual package name, if not, set it to product name.
     this.publishName =
       options.publishName || this.naming.productName.toKebabCase();
+    this.loggingName = this.publishName.replace(/^@google-cloud\//, '');
```

Comment:
this can vary greatly (i.e., googleshopping, google-ads, google-cloud, googleapis, etc.), I think probably best just to use a capturing group after the first backslash